test/tests.js

```diff
@@ -949,4 +949,30 @@ describe('pasync', function() {
 			done();
 		}).catch(done);
 	});
+
+	it('retry', function(done) {
+		var responseArray = [];
+		var retry = pasync.retry(3, function() {
+			responseArray.push('a');
+			responseArray.push('b');
+			responseArray.push('c');
+			return Promise.resolve();
+		}).then(function() {
+			expect(responseArray).to.deep.equal(['a', 'b', 'c']);
+			done();
+		}, function(err) {
+			throw(err)
+		}).catch(done);
+	});
+
+	it('retry with error', function(done) {
+		var retry = pasync.retry(3, function() {
```

Comment:
Add something to this test to make sure it executes this 3 times (maybe a counter).